cypress/integration/regressions.spec.js

```diff
@@ -99,14 +99,6 @@ context('Regressions', () => {
       .should('contain', 'atâhk')
   })
 
-  // https://github.com/UAlbertaALTLab/cree-intelligent-dictionary/issues/176
-  describe('should show idiomatic lemmas', () => {
-    it('The Cree word ayiwinis should give you ayiwinisa as lemma', () => {
-      cy.visitSearch('ayiwinis')
-      cy.get('[data-cy=search-results]')
-        .should('contain', 'ayiwinisa')
-    })
-  })
```

Comment:
commented earlier these tests are not needed anymore

https://github.com/UAlbertaALTLab/cree-intelligent-dictionary/pull/323#discussion_r390724428
https://github.com/UAlbertaALTLab/cree-intelligent-dictionary/pull/323#discussion_r390724841